### PR TITLE
Made new header with all exact parameters and sizes

### PR DESCRIPTION
**Pull request by andriihavryliak**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -14,27 +15,30 @@ body {
 .logo {
     margin: 20px 10px;
 }
+.social-icons{
+    margin: -40px 202px;
```
Comment (vitaliipasiuta) [inline style.css]: Are you sure that negative value is correct solution?

File: style.css [hunk 2]
```diff
@@ -49,73 +53,103 @@ body {
...
     display: block;
-    left: 175px;
-    top: 5px;
+    left: 171px;
+    top: 0px;
 }
-
-.register {
-    margin: 20px;
+.register{
```
Comment (vitaliipasiuta) [inline style.css]: `register` what? Maybe better `register-block` or `register-form` etc?

File: style.css [hunk 3]
```diff
@@ -49,73 +53,103 @@ body {
...
+.register p{
+    display: block;
+    margin: -16px 992px;
+    color: #3783af;
+    font-family: Arial, sans-serif;
     font-size: 13px;
-    color: #1E90FF;
+}
+.register-block {
+    margin: 5px 1006px;
```
Comment (vitaliipasiuta) [inline style.css]: margin 1006 px 
o_O
smth wrong. 
It shouldn't be like this

File: style.css [hunk 4]
```diff
@@ -49,73 +53,103 @@ body {
...
+    text-align: center;
+    font-size: 12px;
+    font-family: Arial;
+    padding: 0 7px 0 6px;
+    letter-spacing: 1px;
+    background: url("img2/nav-line.png") no-repeat right 11px;
+    padding-left: 10px;
+    padding-right: 22px;
+}
+.navigation > ul> li > a > img{
```
Comment (vitaliipasiuta) [inline style.css]: this path is too long.
what if you need to change the structure?
it's better to create few new classes instead ` > ul> li > a > img`

**Review by vitaliipasiuta — CHANGES_REQUESTED** [review]
please, check my comments

File: style.css (hunk 1 above)
Comment (andriihavryliak) [reply style.css]: Small negative values are not so critical

File: style.css (hunk 2 above)
Comment (andriihavryliak) [reply style.css]: register-block I also have))) it's parent block and I think it's correct name

File: style.css (hunk 3 above)
Comment (andriihavryliak) [reply style.css]: OK, I will fix it

File: style.css (hunk 4 above)
Comment (andriihavryliak) [reply style.css]: you're right, I will give this image a class

**Review by vitaliipasiuta — APPROVED** [review]

**Pull request merged**